production/helm/loki/values.yaml

```diff
@@ -1062,7 +1062,7 @@ tracing:
 # Configuration for `minio` child chart
 # -------------------------------------
 minio:
-  enabled: false
+  enabled: true
```

Comment:
I think the Helm chart should be production ready.

Reply:
I am ok with this change, though we need to make sure to change the necessary `values.yaml` in the upgrade guide in the README for people coming from `grafana/loki` who will not have had a minio.

```
minio:
  enabled: false
```

Reply:
Hmm, actually, on second thought, we do not recommend minio for production. so I think having it off by default might be more "production ready" as it's assuming they're providing s3/gcs/azure credentials. WDYT?

Reply:
Oh, I didn't know we don't recommend it. Ok, I'll change this back.